spec/jstests-spec.js

```diff
@@ -337,29 +339,32 @@ describe("Node search", function() {
     document.body.appendChild(query);
   });
 
-  it("should be a successful node search", function() {
```

Comment:
I think keeping the "node search" is a more descriptive test description!

Reply:
when the tests run it automatically prepends the string from the describe() function, so when the tests are run it has the full name
![image](https://user-images.githubusercontent.com/44515357/87994917-fc063780-caa2-11ea-98fc-8cd9d0837b12.png)

Reply:
Oh interesting, I didn't know it prepends the describe() string as well. Good find!